stylua.toml

```diff
@@ -0,0 +1,6 @@
+column_width = 120
```

Comment:
These look like good settings I think (though I'm no expert at stylua).

I like this

Reply:
Great